src/components/BackendExportEntitiesForm.tsx

```diff
@@ -0,0 +1,257 @@
+      marginTop: "1em",
+      padding: "0.5em 0.8em",
+      whiteSpace: "pre-line",
+      height: "400px",
+      overflowY: "scroll"
+    },
+    executionNavHeader: {
+      marginTop: "10px",
+      marginBottom: "10px",
+      padding: "10px",
+      fontSize: "larger"
+    },
+    executionNavHeaderText: {
+      display: "inline-block",
+    },
+    cronContainer: {
+      marginTop: "10px",
+      fontSize: "16px",
+      fontFamily: theme.typography.fontFamily,
+      letterSpacing: "normal",
+    },
+    enableLabel: {
+      fontFamily: theme.typography.fontFamily,
+      fontSize: "16px",
+      letterSpacing: "normal",
+      marginLeft: "-16px",
+    },
+    saveButton: {
+      marginTop: "20px"
+    },
+    progress: {
+      marginTop: "20px"
+    },
+    expandIcon: {
+      color: theme.palette.text.primary
+    },
+    paginationLink: {
+      fontFamily: theme.typography.fontFamily,
+      fontSize: "17px",
+      marginTop: "-4px",
+      letterSpacing: "normal",
+    }
+  };
+});
+
+export default function BackendExportEntitiesForm() {
+  const opslevelPluginApi = useApi(opslevelPluginApiRef);
+  const [configuration, setConfiguration] = useState<AutoSyncConfiguration | null>(null);
+  const [configurationSaving, setConfigurationSaving] = useState<boolean>(false);
+  const [dirty, setDirty] = useState<boolean>(false);
+  const [currentRun, setCurrentRun] = useState<AutoSyncExecution | null>(null);
+  const [currentRunIndex, setCurrentRunIndex] = useState(0);
+  const [runCount, setRunCount] = useState<number | null>(null);
+  const [configExpanded, setConfigExpanded] = useState(false);
+
+  const outputRef = useRef<any>();
+  const currentIndexRef = useRef<number>();
+  useEffect(() => { currentIndexRef.current = currentRunIndex }, [currentRunIndex]);
+  const currentRunRef = useRef<AutoSyncExecution | null>();
+  useEffect(() => { currentRunRef.current = currentRun }, [currentRun]);
+
+  const setSchedule = (schedule: string) => {
+    if(configuration === null) return;
+    if(schedule !== configuration.auto_sync_schedule) {
+      setConfiguration({ auto_sync_enabled: configuration.auto_sync_enabled, auto_sync_schedule: schedule });
+      setDirty(true);
+    }
+  };
+
+  const setEnabled = (enabled: boolean) => {
+    if(configuration === null) return;
+    setConfiguration({ auto_sync_enabled: enabled, auto_sync_schedule: configuration.auto_sync_schedule });
+    setDirty(true);
+  };
+
+  const loadRun = (index = currentRunIndex) => {
+    opslevelPluginApi.getAutoSyncExecution(index)
+      .then((ret) => {
+        const contentChanged = ret.rows.length > 0 && ret.rows[0].output !== currentRunRef?.current?.output;
+        setCurrentRun(ret.rows.length > 0 ? ret.rows[0] : null);
+        setRunCount(parseInt(ret.total_count, 10));
+        setCurrentRunIndex(index);
+        if (contentChanged) {
+          outputRef.current?.scroll?.({ top: outputRef.current.scrollHeight, behavior: 'smooth' });
```

Comment:
Neat! I'm not sure where this `.scroll()` comes from but it works great! I was also not aware the `?.` syntax worked before function arguments. TIL!

Reply:
same lol. the .current gives you the HTML element and that has scroll() out of the box